vnext/src/Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -1198,21 +1198,27 @@ const ExportedForwardRef: React.AbstractComponent<
   React.ElementConfig<typeof InternalTextInput>,
   React.ElementRef<HostComponent<mixed>> & ImperativeMethods,
 > = React.forwardRef(function TextInput(
-  props,
+  {
+    allowFontScaling = true,
+    rejectResponderTermination = true,
+    underlineColorAndroid = 'transparent',
+    ...restProps
```

Comment:
I don't see restProps defined...

Reply:
This pattern is a bit confusing, but this seems to be doing a destructuring assignment of the incoming props parameter, to split out and override a few props, but leave the rest past in into a `restprops` object.

Agree this is pretty opaque. This is how upstream implemented it though.

Reply:
alhhh, sorry... Missed it was an LHS expression as a parameter definition. When reading the code it read to me as argument expression to function call...

I guess the standard prettier is not doing us favors here :D